packages/Operators/src/SharedDomain/DTK_L2ProjectionOperator.hpp

```diff
@@ -114,8 +115,8 @@ class L2ProjectionOperator : virtual public MapOperator
     void applyImpl(
         const TpetraMultiVector &X, TpetraMultiVector &Y,
         Teuchos::ETransp mode = Teuchos::NO_TRANS,
-        double alpha = Teuchos::ScalarTraits<double>::one(),
-        double beta = Teuchos::ScalarTraits<double>::zero() ) const override;
+        Scalar alpha = Teuchos::ScalarTraits<Scalar>::one(),
+        Scalar beta = Teuchos::ScalarTraits<Scalar>::zero() ) const override;
```

Comment:
For the future reference, `ScalarTraits` does not work on a device, will need to use the Kokkos version.